views/partials/sign_up_ffxm.hbs

```diff
@@ -0,0 +1,21 @@
+ <div id="subscribe-to-ffxm" class="modal-content">
+  <h3 class="demi">Sign Up for Firefox Monitor</h3>
+        <p>Firefox Monitor is a service that collects information about data breaches and alerts users when their online accounts are exposed to hackers.</p>
+        <button id="subscribe-fxa-btn" class="button sign-in-btn cell">Subscribe with Firefox Account</button>
+          <form action="/user/add" id="subscribe-form" class="form-group" method="post" novalidate data-no-csrf>
+            <div class="input-group">
+              <input id="subscribe-email-input" class="input-group-field" type="email" name="email" placeholder="Subscribe with Email Account" minlength="1" aria-label="Enter Email Address" tabindex="0" autocomplete="off" />
+              <span id="invalid-signup-email-message" class="error-message">Must be a valid email.</span>
+            </div>
+            <div class="checkbox-group">
+              <input id="additional-emails-checkbox" type="checkbox" name="additional-emails" value="additional-emails" />
+              {{!-- <div class="visible-checkbox-element checkbox"></div> --}}
+              <label>Get the lastest news from Mozilla and Firefox.</label>
```

Comment:
Should this `<label>` be wrapping the checkbox input, or have some `<label for="additional-emails-checkbox">...</label>`.